geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/common/CommonParams.scala

```diff
@@ -62,11 +62,16 @@ trait OptionalDTGParam {
   var dtgField: String = null
 }
 
-trait AttributesParam {
+trait OptionalAttributesParam {
```

Comment:
Looks like the existing use of AttributesParam need to be updated to this new Optional one.